docs/legacy/01-endpoints.md

```diff
@@ -0,0 +1,116 @@
+---
+id: endpoints
+title: Endpoints
+sidebar_label: Endpoints
+slug: /
+---
+
+# AMM DEXes API
+
+## C1: Uniswap Sample
+
+Endpoint: https://old-api.defichain.io/v1/listswaps
+
+Query params:
+
+* network: mainnet (default) / testnet
+
+Sample output:
+```json
+{
+  "ETH_DFI": {
+    "base_id": "1",
+    "base_name": "Ether",
+    "base_symbol": "ETH",
+    "quote_id": "0",
+    "quote_name": "Default Defi token",
+    "quote_symbol": "DFI",
+    "last_price": "1014.38963182",
+    "base_volume": "*",
+    "quote_volume": "*"
+  },
+  "BTC_DFI": {
+    "base_id": "2",
+    "base_name": "Bitcoin",
+    "base_symbol": "BTC",
+    "quote_id": "0",
+    "quote_name": "Default Defi token",
+    "quote_symbol": "DFI",
+    "last_price": "34734.50763876",
+    "base_volume": "*",
+    "quote_volume": "*"
+  },
+  "USDT_DFI": {
+    "base_id": "3",
+    "base_name": "Tether USD",
+    "base_symbol": "USDT",
+    "quote_id": "0",
+    "quote_name": "Default Defi token",
+    "quote_symbol": "DFI",
+    "last_price": "1.60368329",
+    "base_volume": "*",
+    "quote_volume": "*"
+  }
+}
+```
+
+## C3: Yield Farming Sample
+
+Endpoint: https://old-api.defichain.io/v1/listyieldfarming
+
+Query params:
+* network: mainnet (default) / testnet
+
```

Comment:
Probably should provide the typescript interface as well for both? 

Something like this:  https://jellyfish.defichain.com/ocean/address#typescript-definitions